src/uint256.cpp

```diff
@@ -20,10 +20,15 @@ base_blob<BITS>::base_blob(const std::vector<unsigned char>& vch)
 template <unsigned int BITS>
 std::string base_blob<BITS>::GetHex() const
 {
-    char psz[sizeof(data) * 2 + 1];
-    for (unsigned int i = 0; i < sizeof(data); i++)
-        sprintf(psz + i * 2, "%02x", data[sizeof(data) - i - 1]);
-    return std::string(psz, psz + sizeof(data) * 2);
+    static const char *hex_chars = "0123456789abcdef";
+    std::string retval;
+    retval.reserve(sizeof(data) * 2);
+    for (unsigned int i = 0; i < sizeof(data); i++) {
+        uint8_t value = data[sizeof(data) - i - 1];
+        retval.push_back(hex_chars[value >> 4]);
+        retval.push_back(hex_chars[value & 0xf]);
```

Comment:
nit: maybe emplace rather than push (potentially saves an initialization)
@dcousens can you not delete your feedback? I had to dig in my emails to see that I wasn't repeating your feedback.